kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/ProcessorListener.java

```diff
@@ -89,11 +96,11 @@ public void add(Notification<T> obj) {
   }
 
   public void determineNextResync(ZonedDateTime now) {
-    this.nextResync = now.plusSeconds(this.resyncPeriod);
+    this.nextResync = now.plusNanos(this.resyncPeriodInMillis * 1000000);
```

Comment:
Can't really understand what's going on here.
We multiply by 1,000,000 a millisecond unit and add it as a nanosecond?
(it would _probably_ make sense if we divided instead of multiply, but then why not just do `now.plus(resyncPeriodInMillis, ChronoUnit.MILLIS)`)